architecture/service_catalog/index.adoc

```diff
@@ -0,0 +1,191 @@
+<4> A user chooses a service class and requests a new _instance_ of their own. For
+example, a service instance may be a BestDataBase instance named `my_db`.
+<5> A user links, or _binds_, their service instance to a set of pods (their
+application). For example, the `my_db` service instance may be bound to the
+user's application called `my_app`.
+
+This infrastructure allows a loose coupling between applications running in
+{product-title} and the services they use. This allows the application that uses
+those services to focus on its own business logic while leaving the management
+of these services to the provider.
+
+[[service-catalog-concepts-terminology]]
+== Concepts and Terminology
+
+Service Broker::
+A _service broker_ is a server that conforms to the OSB API specification. The
+software could be hosted within your own {product-title} cluster or elsewhere.
++
+An API resource representation of a service broker is registered with the
+{product-title} cluster, available via web endpoint, that manages a set of one
+or more services. These are created and managed by cluster administrators who
+wish to use that service broker to make new types of managed services available
+within their cluster.
++
+A resource of this type encapsulates connection details for that service broker
+and specifies the set of services (and variations of those services) to
+{product-title} that should then be made available to users.
++
+.Example `Broker` Resource
+----
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Broker
+metadata:
+  name: BestDataBase
+spec:
+  url: http://bestdatabase.example.com
+----
+
+Service Class::
+Also synonymous with "service" in the context of the service catalog, a _service
+class_ is a type of managed service offered by a particular broker. Each time a
+new broker resource is added to the cluster, the service catalog controller
+connects to the corresponding service broker to obtain a list of service
+offerings. A new `ServiceClass` resource is automatically created for each.
++
+[NOTE]
+====
+{product-title} also has a core concept called
+xref:../../architecture/core_concepts/pods_and_services.adoc#services[services],
+which are separate Kubernetes resources related to internal load balancing.
+These resources are not to be confused with how the term is used in the context
+of the service catalog and OSB API.
+====
++
+.Example `ServiceClass` Resource
+----
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: ServiceClass
+metadata:
+  name: smallDB
+  brokerName: BestDataBase
+  plans: [...]
+----
+
+Service Plan::
+A _service plan_ is one type of variant of a service class. For example, a
+service class may expose a set of plans that offer varying degrees of
+quality-of-services (QoS), each with a different cost associated with it.
+
+Service Instance::
+A _service instance_ is a provisioned instance of a service class. These are
+created by cluster users who wish to make a new, distinct instance of some type
```

Comment:
This kind of implies that if you want to share an instance somehow, you don't need to make a new instance.  I would just say something like "When a user wants to use the capability provided by a ServiceClass, they make a new Instance"